fastlane_bot/tools/pool_shutdown.py

```diff
@@ -182,92 +182,9 @@ def generate_shutdown_tx(self, tkn: str) -> str:
             self.tx_helpers.get_max_priority_fee_per_gas_alchemy()
             * self.mgr.cfg.DEFAULT_GAS_PRICE_OFFSET
         )
-        transaction = self.build_transaction(
-            tkn=tkn,
-            gas_price=gas_price,
-            max_priority_gas=max_priority_gas,
-            nonce=self.tx_helpers.get_nonce(),
-        )
+        transaction = self.tx_helpers.build_transaction_generic(self.bancor_network_contract.functions.withdrawPOL, self.mgr.web3.to_checksum_address(tkn), gas_price=gas_price, max_priority_fee=max_priority_gas, nonce=self.tx_helpers.get_nonce())
```

Comment:
Please format this into something readable.

Reply:
Updated this - please review & close if satisfactory.